### PR TITLE
fix cra for altref

### PR DESCRIPTION
**Pull request by lijing0010**
Signed-off-by: Jing Li <jing.b.li@intel.com>

### CODE REVIEW

**Comment by lijing0010** [comment]
Current altref for CRA is disabled(due to some bugs), this PR will enable altref for cra, together with SCD cases

**Comment by hassount** [comment]
@joelsole @mafonsoN @andrey-norkin @anaghdin any issues with this ?

**Comment by anaghdin** [comment]
The changes look fine, but can we get the bdrate impact?

File: Source/Lib/Common/Codec/EbPictureDecisionProcess.c [hunk 1]
```diff
@@ -3856,12 +3856,24 @@ void* picture_decision_kernel(void *input_ptr)
...
                                     picture_control_set_ptr->temp_filt_pcs_list[pic_itr] = pcs_itr;
                                 }
+
+#else
+                                //get previous+current pictures from the the pre-assign buffer
+                                for (int pic_itr = 0; pic_itr <= num_past_pics; pic_itr++) {
+                                    PictureParentControlSet* pcs_itr = (PictureParentControlSet*)encode_context_ptr->pre_assignment_buffer[pictureIndex - num_past_pics + pic_itr]->object_ptr;
+                                    picture_control_set_ptr->temp_filt_pcs_list[pic_itr] = pcs_itr;
+                                }
+#endif
```
Comment (mafonsoN) [inline Source/Lib/Common/Codec/EbPictureDecisionProcess.c]: It seems like the only change is in the num_past_pics. Can you move this up where num_past_pics is first assigned?

**Pull request merged**